### PR TITLE
Sw lib guide

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick @addyosmani @jpmedley 

This adds Joe's doc to the repo.

### CODE REVIEW

**Comment by jeffposnick** [comment]
I left some feedback on Joe's draft doc recently. Might be best to hold off on merging this one until the source stabilizes.

**Comment by jpmedley** [comment]
Or you can pull it completely. I had intended to share the doc for the hackathon and do the conversion after.

File: docs/getting-started/sw-lib.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+* sw-lib.js
+* sw-lib.js.map
+* sw-lib.min.js
+* sw-lib.min.js.map
+
+You can do this with a simple command like:
+
+```
+cp node_modules/sw-lib/build/* app/third-party/sw-lib/
```
Comment (jpmedley) [inline docs/getting-started/sw-lib.md]: Matt,

In my experiments, I originally copied the files to a subdirectory but found that some functionality didn't work. I think (if I remember correctly) that precaching worked but runtime didn't. That's when I slapped my head and said, "Service workers, duh!." I moved it to the root and everything was fine.

Is there something I overlooked?

@jeffposnick

**Comment by gauntface** [comment]
@jpmedley over to you.

**Pull request closed without merging**